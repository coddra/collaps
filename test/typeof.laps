print(typeof (1 == 1))
print(typeof .1)
print(typeof 1)
print(typeof "")
print(typeof [])
print(typeof typeof 1)
