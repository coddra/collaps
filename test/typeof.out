{ parent: undefined, fields: [] }
{ parent: undefined, fields: [] }
{ parent: undefined, fields: [] }
{ parent: undefined, fields: [] }
{ parent: { parent: undefined, fields: [] }, fields: [{ name: count, readonly: true }, { name: capacity, readonly: true }, { name: readonly, readonly: true }] }
{ parent: { parent: undefined, fields: [] }, fields: [{ name: parent, readonly: true }, { name: fields, readonly: true }] }
